src/main/webapp/map.html

```diff
@@ -0,0 +1,125 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Hello Google Maps</title>
+    <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyBZkWQmagGmCHKRX2osG0BSWiUYLOZaxWA"></script>
+    <script>
+        function createMap(){
+            const map = new google.maps.Map(document.getElementById('map'), {
+
+            //Google Plex offic
+            center: {lat: 37.422, lng: -122.084},
+            zoom: 16,
+
+            //Styles map in Night Mode
+            styles: [
+                {elementType: 'geometry', stylers: [{color: '#242f3e'}]},
```

Comment:
So here do you have a style for everything and more specific styles for certain types of features? And the element types are not already set to a certain styler by the first section of styles?

Reply:
Um I not really sure, I just know this styles it in night mode. And I think it changes the color of the roads